Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -97,14 +97,98 @@ public static Command DeleteActionMap(int actionMapIndex)
             };
         }
 
+        public static Command CopyActionMapSelection()
```

Comment:
Wouldn't having these as Commands imply that they become undo steps? I do not believe we should have read-only operations such as Copy as undo steps?

Reply:
As the SerializedObject does not change and we don't modify the InputActionsEditorState no Undo operation is recorded for Copy. Cutting and pasting items is undoable (what I think feels natural). 
In terms of the logic structure of the code, it is a good question if separating the Copy Methods from the rest because they are not undoable, is a good idea. But from my subjective perspective I would say we are good to leave them there as Copy is a Command either way.

Reply:
Allright, thanks for explaining, then I think we are good as is.